packages/devtools_app/lib/src/screens/vm_developer/vm_service_private_extensions.dart

```diff
@@ -897,15 +899,15 @@ extension CpuSamplesPrivateView on CpuSamples {
   }
 
   List<ProfileCode> get codes {
-    var _codes = _expando[this];
-    if (_codes == null) {
-      _codes = json![_kCodesKey]
+    var codes = _expando[this];
```

Comment:
can we rename something other than `codes` to avoid confusion with the parent getter? `codes_`, `cachedCodes`, `codesFromExpando`, etc.